charts/kube-prometheus-stack/templates/prometheus-operator/certmanager.yaml

```diff
@@ -18,7 +18,7 @@ metadata:
   namespace: {{ template "kube-prometheus-stack.namespace" . }}
 spec:
   secretName: {{ template "kube-prometheus-stack.fullname" . }}-root-cert
-  duration: 43800h0m0s # 5y
+  duration: {{ .Values.prometheusOperator.admissionWebhooks.certManager.rootCert.duration }}
```

Comment:
how about we add the template for the duration and if not present make the default as 5y. In this way, we have support for both and folks don't need to edit the default values.yml file until they desired to change the duration.

Does that sound reasonable to you?

Reply:
Very reasonable, will do it soon

Reply:
Updated